prober/http.go

```diff
@@ -29,25 +31,34 @@ var (
 		},
 		[]string{"name", "status_code", "result", "url"},
 	)
+	dnsLookupDurations = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "health_http_dns_lookup_time_seconds",
+			Help:    "The response time of dns lookup",
+			Buckets: []float64{0.001, 0.005, 0.01, 0.02, 0.03, 0.05, 0.075, 0.1, 0.2, 0.5, 0.75, 1, 1.5, 2, 2.5, 3, 4, 5},
+		},
+		[]string{"name", "status_code", "result", "url"},
```

Comment:
please add another label for dns_lookup_status or _error

Reply:
done